internal/rule/not_regex.go

```diff
@@ -0,0 +1,85 @@
+package rule
+
+import (
+	"fmt"
+	"regexp"
+	"sync"
+)
+
+type NotRegex struct {
+	name         string
+	exclusive    bool
+	regexPattern string
+	*sync.RWMutex
+}
+
+func (rule *NotRegex) Init() Rule {
+	rule.name = "not_regex"
+	rule.exclusive = false
+	rule.RWMutex = new(sync.RWMutex)
+
+	return rule
+}
+
+func (rule *NotRegex) GetName() string {
+	rule.RLock()
+	defer rule.RUnlock()
+
+	return rule.name
+}
+
+// 0 = regex pattern
+func (rule *NotRegex) SetParameters(params []string) error {
+	rule.Lock()
+	defer rule.Unlock()
+
+	if len(params) == 0 {
+		return fmt.Errorf("regex pattern not exists")
```

Comment:
```suggestion
		return errors.New("regex pattern not exists")
```

There is no formating here